### ISSUE TITLE
config_option shortcode is escaping HTML

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#2489]
Visit https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#peer-service-name and you'll see:

> <img width="695" alt="Screen Shot 2023-03-12 at 20 17 37" src="https://user-images.githubusercontent.com/4140793/224583189-9d6185d0-b4ef-46cb-bb29-033a3ead070a.png">

It seems that Hugo's `trim` function is escaping HTML.

**Comment by shyamtawli** [comment]
Hey can I work on it.

**Comment by chalin** [comment]
No worries! It does look like a quick-fix candidate.

**Comment by shyamtawli** [comment]
No Issue @svrnm 
I will see other beginner issues.

**Comment by chalin** [comment]
Thanks! For the record, it seems likely that Go templates are escaping based on context, similar to what happened in #2509, which was resolved via:

- #2511

It's not clear to me why this is happening for a shortcode rendered as markdown.

**Comment by vitorvasc** [comment]
@chalin @svrnm is this issue still valid?

**Comment by svrnm** [comment]
I don't know tbh, we changed a lot of content and I can not see a single instance of this, so maybe we can close it and open a new one if this is still a thing?

**Comment by vitorvasc** [comment]
I agree! Closing this issue, then. I'll keep an eye out to see if this ever pops up again.
